assets/worker/0700_worker_daemonset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nfd-worker
+  name: nfd-worker
+spec:
+  selector:
+    matchLabels:
+      app: nfd-worker
+  template:
+    metadata:
+      labels:
+        app: nfd-worker
+    spec:
+      nodeSelector:
+        node-role.kubernetes.io/node: ""
```

Comment:
Not a label in upstream K8s. Should be dropped(?)

Reply:
well, but then how can we target the nfd-workers to the right nodes?

Reply:
Just deploy on all schedulable nodes(?)

Reply:
you think we should then also drop 
```yaml
 18       tolerations:
 19       - operator: Exists
 20       - key: nvidia.com/gpu
 21         operator: Exists
 22         effect: NoSchedule
```

Reply:
or leave it like
```yaml
 16       nodeSelector:
 18       tolerations:
 19       - operator: Exists
 20       - key: nvidia.com/gpu
 21         operator: Exists
 22         effect: NoSchedule
```